tsconfig.json

```diff
@@ -10,7 +10,7 @@
         "declaration": false,
         "noImplicitAny": false,
         "noImplicitUseStrict": false,
-        "noUnusedLocals": true,
+        "noUnusedLocals": false,
```

Comment:
>"noUnusedLocals": false, [](start = 8, length = 24)

why? if you really need an unused local (I'm assuming the window.clarity from globalize?) we should just be disabling the rule for that line.

Reply:
No, this is actually for something in our 3rd party compress/decompress code, which we technically can't touch. And also, unlike tslint, you can't disable the tsconfig rule per-line.

---
In reply to: [250700312](https://github.com/Microsoft/clarity-js/pull/126#discussion_r250700312) [](ancestors = 250700312)